ci-operator/jobs/infra-periodics.yaml

```diff
@@ -1,4 +1,32 @@
 periodics:
+- agent: kubernetes
+  labels:
+    ci.openshift.io/role: infra
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  cluster: ci/api-build01-ci-devcluster-openshift-com:6443
+  cron: ""
+  decorate: true
+  interval: 1m
+  name: periodic-ci-image-import-to-build01
+  skip_cloning: true
+  spec:
+    containers:
+    - args:
+      - -c
+      - |
+        set -o errexit
+        set -o nounset
+        set -o pipefail
+        for is in $(oc get is -n ci -o=jsonpath='{.items[*].metadata.name}'); do
+          echo "importing ${is} ..."
+          oc import-image -n ci "${is}" --all
```

Comment:
I don't think this does what you think it will do, it will import IS from the `ci` namespace on the build01 cluster to whatever namespace the job is running in on the build01 cluster. The intention is to import the IS from the ci.api cluster or not?

In that case we ne need two different kubeconfigs here and to tell the `import-image` command to do it from a different cluster.

Reply:
> I don't think this does what you think it will do, it will import IS from the `ci` namespace on the build01 cluster to whatever namespace the job is running in on the build01 cluster. The intention is to import the IS from the ci.api cluster or not?

The intention is to import the image (tags) of the `imagestreams` (defined in build01) on ci namespace to build01. We do this because the default scheduled importing is not quick enough (every 15m) and we cannot find where to set this interval up.

```
oc get is -n ci ci-ns-ttl-controller -o yaml | yq -y -r '.spec.tags[].from'
kind: DockerImage
name: registry.svc.ci.openshift.org/ci/ci-ns-ttl-controller:latest
```

The above example defines a tag of ci-ns-ttl-controller which imports `registry.svc.ci.openshift.org/ci/ci-ns-ttl-controller:latest` (on api.ci).


> In that case we ne need two different kubeconfigs here and to tell the `import-image` command to do it from a different cluster.

This job has to run on `build01` for now (as its cluster defines).
If we want to support it to run on `api.ci`, we need to supply `--kubeconfig`.
But i do not think we need to do that right now.